mods/ctf/ctf_chat/init.lua

```diff
@@ -256,6 +256,10 @@ minetest.register_chatcommand("t", {
 				chatplus.log("<" .. name .. "> ** ".. param .. " **")
 			end
 
+			if param == "" then
+		return false, "-!- Empty team message, see /help t"
+			end
```

Comment:
Please adjust your code here.
```suggestion
			if param == "" then
		        return false, "-!- Empty team message, see /help t"
			end
```